Makefile

```diff
@@ -43,7 +43,7 @@ test:
 	$(MAKE) migrations-test
 
 load-fixtures:
-	docker-compose -f docker-compose.test.yml run --rm -w /go/src/github.com/CMSgov/bcda-ssas-app/db/migrations tests migrate -database "postgres://postgres:toor@db:5432/bcda?sslmode=disable" -path ./ up
+	docker-compose -f docker-compose.test.yml run --rm migrate /migrate -database "postgres://postgres:toor@db:5432/bcda?sslmode=disable" -path db/migrations/ up
```

Comment:
I might be overlooking some of the in-depth details.  I wonder if this PR can be simplified to two lines of code in the `Makefile`.  No changes anywhere else:
```
docker pull migrate/migrate
docker run --rm migrate/migrate -database "postgres://postgres:toor@db:5432/bcda?sslmode=disable" -path db/migrations/ up
```

Reply:
Might have to mount a volume so the container can see `db/migrations`.  Here is an example where we do that: https://github.com/CMSgov/bcda-ssas-app/blob/master/Makefile#L11

Reply:
Good advice!  I also had to specify the network.